observatory-dags/observatory/dags/telescopes/scopus.py

```diff
@@ -524,7 +524,7 @@ def retrieve(self, query):
 
```

Comment:
It would be good for this function (and others, it looks like most have type hints though) to use type hints to make it clear what the input and return types are, e.g. (not sure if these are correct).
```python
def retrieve(self, query: str) -> Tuple[List, int, int]:
```